tests/hawc/apps/common/dynamic_forms/test_forms.py

```diff
@@ -0,0 +1,82 @@
+from copy import deepcopy
+
+import pytest
+from crispy_forms.utils import render_crispy_form
+from pytest_django.asserts import assertInHTML
+
+from hawc.apps.common.dynamic_forms import Schema
+
+
+class TestDynamicForm:
+    def test_field_rendering(self, complete_schema):
+        # ensure schema with all field types can render without error
+        schema = Schema.parse_obj(complete_schema)
+        form_rendering = render_crispy_form(schema.to_form({}))
+        assert len(form_rendering) > 0
+
+    @pytest.mark.skip
+    def test_yesno_rendering(self, complete_schema):
```

Comment:
this test is failing, but fixing it would be out of scope for this PR since it's related to rendering a form from its schema. Should I remove the tests from this PR?

Reply:
I just fixed the test.